src/main/java/wolox/training/models/Book.java

```diff
@@ -67,6 +68,18 @@ public Book() {
         //Constructor for JPA
     }
 
+    public Book(BookInfoDTO bookInfoDTO) {
+        this.genre = "N/A - External API";
```

Comment:
I think you can map this attribute to the `subjects` element in the API response. You can pick the first one, for example :)